packages/synapse-react-client/src/components/CardContainerLogic/CardContainerLogic.stories.ts

```diff
@@ -55,7 +55,7 @@ export const EmptyResults: Story = {
 
 export const ObservationCard: Story = {
   args: {
-    sql: `SELECT "Observation Submitter Name" as "submitterName", Synapse_id as "submitterUserId", "Observation Time" as "time", "Observation Time Units" as "timeUnits", "Observation Text" as "text", "Observation Type" as "tag" FROM syn26344832 WHERE "Observation Time" IS NOT NULL`,
```

Comment:
[This story](https://sage-bionetworks.github.io/synapse-web-monorepo/?path=/story/explore-cardcontainerlogic--observation-card) was broken due to a resource does not exist error for `syn26344832`